packages/ui-extensions-sdk/src/types.ts

```diff
@@ -341,3 +341,7 @@ export interface DeprecatedEventUsage {
 }
 
 export type DeprecatedUsage = DeprecatedEventUsage; // we can union type here later
+
+export interface Config {
+    [key: string]: string | boolean;
+}
```

Comment:
Should we call this `OrgConfig` to be consistent + safe? It's likely user config would also be the same thing but we don't know for sure, we can always union-type later

Reply:
Ah oops; yeah — must've missed this one

Reply:
Alright, this should be updated